python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -1126,6 +1126,19 @@ def get_defer_args(self) -> Sequence[str]:
 
         return ["--defer", state_flag, self.state_path]
```

Comment:
i opted for `--defer-state` here since it seemed appropriate to use the more scoped flag